independent-publisher-connectors/AzureFunctionApps/Readme.md

```diff
@@ -0,0 +1,17 @@
+# Azure Function Apps Connector for Application Registrations
```

Comment:
@Stephan-Rijk please include all required sections in the readme file. please follow our template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)and for raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md).